src/apps/sources/__test__/pages/CreateSourcePage.test.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import CreateSourcePage, {mapActionsToProps, mapStateToProps} from "../../pages/CreateSourcePage";
+import {APIOrg, APIProfile} from "../../../authentication";
+import {APISource, Source} from "../../types";
+import {createSourceAction, createSourceDispatchAction} from "../../redux";
+import {act, fireEvent, render} from "@testing-library/react";
```

Comment:
can we import this from `src/test-utils.tsx` hoc. That way we don't have import `store`, `provide`, `router`

Reply:
@rakesh-1010 Done !